### PR TITLE
Added CourseDetailsTable Component and Test

### LINKED ISSUE #14: Priority: Table Component, test, and story
- Create a table component that displays two columns; one for the Enroll code, and one for Quarter in QYY format. 
- This table will be displayed on the CourseDetails page.
- No need to add the rest of the fields, that data will be added in issue #23

### PR DESCRIPTION
**Pull request by ivan512az**
In this PR, Mariana and I added the corresponding EnrollmentCode and Quarter table to the CourseDetailsTable.

The image on Storybook:
Screenshot 2023-11-27 at 9 16 29 PM

Updated storybook picture:
Screenshot 2023-11-28 at 2 34 16 PM

Closes #14

### CODE REVIEW

File: frontend/src/fixtures/courseDetailsFixtures.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const courseDetailsFixtures = {
+  twoCourses: [
+    {
+      quarter: "20211",
+      enrollmentCode: "ABC123",
```
Comment (pconrad) [inline frontend/src/fixtures/courseDetailsFixtures.js:5]: Can we make the enrollment code fixtures match the format of actual enrollment codes?  Those are 5 digit numbers.
```suggestion
      enrollmentCode: "12345",
```

File: frontend/src/tests/components/CourseDetailsTable.test.js/CourseDetailsTable.test.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      const header = screen.getByTestId(`${testId}-cell-row-0-col-${field}`);
+      expect(header).toBeInTheDocument();
+    });
+
+    expect(
+      screen.getByTestId(`${testId}-cell-row-0-col-quarter`),
+    ).toHaveTextContent("W21");
+    expect(
+      screen.getByTestId(`${testId}-cell-row-1-col-enrollmentCode`),
+    ).toHaveTextContent("CDF456");
```
Comment (pconrad) [inline frontend/src/tests/components/CourseDetailsTable.test.js/CourseDetailsTable.test.js:55]:
```suggestion
    ).toHaveTextContent("23456");
```

**Review by pconrad — CHANGES_REQUESTED** [review]
I'm having trouble reviewing this because there is no context about what problem this solves for the end user.   The issue that this closes is no help either; it looks like this:

image


@KateUnger you wrote this issue.  What larger epic or story is this a part of?  We have that we are adding two fields to a form, but what page is that form being used on ?

Where does this fit in?   When we create or modify a component, we need to understand where that component is going to be used, and I don't see any connection between this form and a proposed usage.  Further, the issue talked about a "form" but this is a table.

Please consult with one another, and clarify both the issue and the PR.  We don't merge code for it's own sake, but only if it's connected to meeting a user need.

**Comment by ivan512az** [comment]
@pconrad Fixed enrollment code to only numbers as stated. Additionally, I fixed the kanban instructions to reflect on the work from this PR.

**Review by pconrad — APPROVED** [review]
LGTM

**Review by ThatsBadJuju — APPROVED** [review]

**Pull request merged**